### PR TITLE
Feature swap send ui changes

### PR DESCRIPTION
**Pull request by bradleySuira**
This branch contains ui changes for the send and confirmation views

image
Screen Shot 2020-11-26 at 10 23 22 AM

### CODE REVIEW

File: src/views/Swap.vue [hunk 1]
```diff
@@ -237,7 +235,7 @@ export default {
     amountError () {
       const amount = BN(this.safeAmount)
 
-      if (amount.gt(this.available)) return 'Amount exceeds available balance.'
+      if (amount.gt(this.available)) return 'Lower amount. This exceeds your available balance.'
```
Comment (monokh) [inline src/views/Swap.vue:238]: Seems to drop into 2 lines :(

**Comment by monokh** [comment]
@bradleySuira are you using some kind of formatter? Some of the templates are very verbose, would be good if you could adjust the settings and also if there is a config commit it so everyone can use it.

File: src/views/Swap.vue [hunk 2]
```diff
@@ -138,7 +137,6 @@ import { dpUI, prettyBalance, prettyFiatBalance } from '@/utils/coinFormatter'
 import { getChainFromAsset, getAssetColorStyle, getAssetIcon } from '@/utils/asset'
 import { shortenAddress } from '@/utils/address'
 import { TX_TYPES, getTxFee } from '@/utils/fees'
-import Warning from '@/components/Warning'
```
[the remark this answers is not in the document]
Comment (bradleySuira) [reply src/views/Swap.vue:141]: fixed

File: src/assets/scss/style.scss [hunk 3]
```diff
@@ -261,17 +262,58 @@ textarea {
...
     height: 75px;
   }
 }
+
+.details-container {
+  text-align: left;
+  margin-bottom: 1rem;
+
+  .details-header {
+    width: 100%;
```
[the remark this answers is not in the document]
Comment (bradleySuira) [reply src/assets/scss/style.scss:285]: fixed

File: src/views/Swap.vue (hunk 1 above)
Comment (bradleySuira) [reply src/views/Swap.vue:238]: Agree, let me as to Thessy about it

**Comment by bradleySuira** [comment]
> @bradleySuira are you using some kind of formatter? Some of the templates are very verbose, would be good if you could adjust the settings and also if there is a config commit it so everyone can use it.

Hi @monokh I'm using the `nom run lint --fix` script before push, I can add it to something like husky pre-commit but I think that I need to fix the VS code settings to be equals than yours.

File: src/views/Swap.vue (hunk 1 above)
Comment (bradleySuira) [reply src/views/Swap.vue]: In this case I need to put more to the left the label to use the message that the want to use

File: src/views/Send.vue [hunk 4]
```diff
@@ -1,32 +1,86 @@
...
+              ><span class="text-muted">Available</span> {{ available }}
+              {{ asset }}</span
+            >
+            <div
+              class="float-right btn-group btn-group-toggle"
+              data-toggle="buttons"
+            >
+              <label
+                class="btn btn-light btn-outline-dark btn-sm"
+                @click="didClickOnMaxAmount"
```
[the remark this answers is not in the document]
Comment (bradleySuira) [reply src/views/Send.vue:59]: got it, I will follow your guidelines, it's fixed now

Thanks!

**Pull request merged**